apps/web/src/utils/verifyBscNetwork.ts

```diff
@@ -1,5 +1,5 @@
 import { ChainId } from '@pancakeswap/chains'
 
-export const verifyBscNetwork = (chainId: number) => {
-  return chainId === ChainId.BSC || chainId === ChainId.BSC_TESTNET
+export const verifyBscNetwork = (chainId?: number) => {
+  return chainId && (chainId === ChainId.BSC || chainId === ChainId.BSC_TESTNET)
```

Comment:
I think it's meant to return boolean? Shall we wrap it up?
```suggestion
  return Boolean(chainId && (chainId === ChainId.BSC || chainId === ChainId.BSC_TESTNET))
```